File: iks-core/src/com/iks/cms/core/sql/projection/Projections.java
package com.iks.cms.core.sql.projection;

import com.iks.cms.core.sql.*;

/**
 * @author Igor Kaynov
 */
public class Projections {
  public static IProjection rowCount(Column column) {
    return new Projection( "count", column );
  }
  public static IProjection rowCount() {
    return new Projection( "count", null );
  }
}

File: iks-core/src/com/iks/cms/core/sql/projection/Projections.kt
package com.iks.cms.core.sql.projection

import com.iks.cms.core.sql.*

/**
 * @author Igor Kaynov
 */
object Projections {
  fun rowCount(column: Column): IProjection {
    return Projection("count", column)
  }

  fun rowCount(): IProjection {
    return Projection("count", null)
  }
}
